### PR TITLE
fix: migrate to modern dependencies

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: example/DeltaBlue.dart [hunk 1]
```diff
@@ -87,19 +88,16 @@ class Strength {
   }
 }
 
-
 // Compile time computed constants.
-const REQUIRED        = const Strength(0, "required");
+const REQUIRED = const Strength(0, "required");
```
Comment (filiph) [inline example/DeltaBlue.dart]: [nit] this could be updated to lowercase constants, but honestly -- let's just land this first.

Comment (kevmoo) [reply example/DeltaBlue.dart]: yup

Comment (rmacnak-google) [reply example/DeltaBlue.dart]: Please do not reformat benchmarks. It makes it harder to compare versions.

Comment (kevmoo) [reply example/DeltaBlue.dart]: Hrm...we standardize on dartfmt for everything – no one has touched this package in years...

Why is this important now?

**Pull request merged**